src/legacy/core_plugins/kibana/public/management/sections/settings/advanced_settings.js

```diff
@@ -170,6 +171,13 @@ export class AdvancedSettings extends Component {
         <EuiSpacer size="m" />
         <CallOuts />
         <EuiSpacer size="m" />
+
+        <AdvancedSettingsVoiceAnnouncement
+          queryText={query.text}
+          settings={filteredSettings}
+          totalCounts={this.categoryCounts}
```

Comment:
Where is `totalCounts` used in `AdvancedSettingsVoiceAnnouncement`?

Reply:
@maryia-lapata nowhere, thank you 🙏